src/main/java/com/openlattice/postgres/ResultSetAdapters.java

```diff
@@ -635,6 +636,17 @@ public static EntitySet entitySet( ResultSet rs ) throws SQLException {
         final var flags = entitySetFlags( rs );
         final var partitions = partitions( rs );
         final var partitionVersion = partitionVersions( rs );
+        final var timeToExpiration = timeToExpiration( rs );
```

Comment:
probably good to break this out into a `public static DataExpiration dataExpiration( ResultSet rs` that's called from `entitySet` for readability / standardization / in case we ever use this object anywhere else